simplified-main/src/main/java/org/nypl/simplified/main/MainServices.kt

```diff
@@ -162,24 +160,15 @@ internal object MainServices {
   private fun initializeDirectories(context: Context): Directories {
     this.logger.debug("initializing directories")
 
-    val directoryPrivateBase =
-      context.filesDir
-    val directoryPrivateBaseVersioned =
-      File(directoryPrivateBase, this.CURRENT_DATA_VERSION)
-    val directoryStorageBase =
-      this.determineDiskDataDirectory(context)
```

Comment:
Is the method `determineDiskDataDirectory ` still used?

Reply:
If it is still used can we at least remove the block referencing external storage?

Reply:
My mistake, I thought I deleted it. Pushed a commit that removes the unused method.